doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,467 @@
+  >>> solution
+      {'message': 'The solver successfully reached the end of the integration interval.', 'nfev': 68, 'njev': 0, 'nlu': 0, 'sol': None, 'status': 0, 'success': True, 't': [ 0.          0.20408163  0.40816327  0.6122449   0.81632653  1.02040816
+        1.2244898   1.42857143  1.63265306  1.83673469  2.04081633  2.24489796
+        2.44897959  2.65306122  2.85714286  3.06122449  3.26530612  3.46938776
+        3.67346939  3.87755102  4.08163265  4.28571429  4.48979592  4.69387755
+        4.89795918  5.10204082  5.30612245  5.51020408  5.71428571  5.91836735
+        6.12244898  6.32653061  6.53061224  6.73469388  6.93877551  7.14285714
+        7.34693878  7.55102041  7.75510204  7.95918367  8.16326531  8.36734694
+        8.57142857  8.7755102   8.97959184  9.18367347  9.3877551   9.59183673
+        9.79591837 10.        ], 't_events': None, 'y': [[1.         0.8588458  0.76037109 0.68865638 0.63448858 0.59298188
+        0.56070716 0.53512394 0.51478622 0.49863364 0.48571594 0.47520885
+        0.4666383  0.45971742 0.45416064 0.44968802 0.44602514 0.4429604
+        0.44043698 0.4383935  0.43676122 0.43546445 0.43442049 0.43353967
+        0.43277066 0.43212494 0.43159916 0.43118497 0.43086892 0.43063257
+        0.43045243 0.43029995 0.43014456 0.42999498 0.42986356 0.42975504
+        0.42967192 0.42961451 0.42958085 0.42956676 0.42956583 0.42956944
+        0.42956671 0.42954545 0.42951959 0.4294984  0.42948121 0.42946735
+        0.42945611 0.4294468 ]
+      [1.         0.9294229  0.88018554 0.84432819 0.81724429 0.79649094
+        0.78035358 0.76756197 0.75739311 0.74931682 0.74285797 0.73760443
+        0.73331915 0.72985871 0.72708032 0.72484401 0.72301257 0.7214802
+        0.72021849 0.71919675 0.71838061 0.71773223 0.71721025 0.71676983
+        0.71638533 0.71606247 0.71579958 0.71559248 0.71543446 0.71531629
+        0.71522621 0.71514997 0.71507228 0.71499749 0.71493178 0.71487752
+        0.71483596 0.71480726 0.71479042 0.71478338 0.71478292 0.71478472
+        0.71478335 0.71477272 0.71475979 0.7147492  0.71474061 0.71473367
+        0.71472805 0.7147234 ]
+      [0.         0.1411542  0.23962891 0.31134362 0.36551142 0.40701812
+        0.43929284 0.46487606 0.48521378 0.50136636 0.51428406 0.52479115
+        0.5333617  0.54028258 0.54583936 0.55031198 0.55397486 0.5570396
+        0.55956302 0.5616065  0.56323878 0.56453555 0.56557951 0.56646033
+        0.56722934 0.56787506 0.56840084 0.56881503 0.56913108 0.56936743
+        0.56954757 0.56970005 0.56985544 0.57000502 0.57013644 0.57024496
+        0.57032808 0.57038549 0.57041915 0.57043324 0.57043417 0.57043056
+        0.57043329 0.57045455 0.57048041 0.5705016  0.57051879 0.57053265
+        0.57054389 0.5705532 ]], 'y_events': None}
+  >>> y = solution.y # Extract the y (concentration) values from SciPy solution result
+  >>> plt.plot(t_eval, y.T) # doctest: +SKIP
+```
+
+```{eval-rst}
+.. plot:: 
```

Comment:
It should be possible to do this directly with myst. The rst directives translate directly into markdown syntax. See https://myst-parser.readthedocs.io/en/v0.13.6/using/syntax.html#directives-a-block-level-extension-point

Reply:
Yeah, I tried, using that format:

````
```{plot}
---
format: doctest 
include-source: True
---
>>> plt.plot([1, 2, 3], [4, 5, 6]) # doctest: +SKIP
```
````

and a slightly different one:

````
```{plot}
:format: doctest 
:include-source: True

>>> plt.plot([1, 2, 3], [4, 5, 6]) # doctest: +SKIP
```
````

and neither gave any output.

I asked on [Write the Docs (Slack for technical writing)](https://www.writethedocs.org/slack/) and the Executable Books lead developer @chrisjsewell replied:

> Yeh so from here: https://github.com/matplotlib/matplotlib/blob/269c0b94b4fcf8b1135011c1556eac29dc09de15/lib/matplotlib/sphinxext/plot_directive.py#L359
> the default `TEMPLATE` is hard-coded to be in rST.
> It looks like you can override this with the `plot_template` configuration option,
> or obviously you can raise a feature request with them to include such a template :sweat_smile:

Which I believe means someone would have to create a `TEMPLATE` which works for Markdown.

Reply:
I actually remember having an issue with plotting in my custom functions guide too. That's good to know. So for now if we want to include plots here, we need to rewrite this page to use RST?

Reply:
No, thankfully it's fine for the page to be in Markdown as long as the plot directive is in RST--[a simple plot block renders a graph image fine](https://output.circle-artifacts.com/output/job/03cc4694-a200-4587-aa2b-bdd4fe6dd463/artifacts/0/doc/_build/html/guides/solving/solve-ode.html#numerically-solve-an-ode-in-scipy) using the wrapper:

````
```{eval-rst}
```
````

I think the remaining critical problem is importing scipy within the plot directive.